src/Persistence/PaYaml/Models/PowerFx/PFxDataType.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.PaYaml.Models.PowerFx;
+
+public enum PFxDataType
```

Comment:
We are linking with 
    <PackageReference Include="Microsoft.PowerFx.Core" Version="1.2.0" />

do we need to recreate Power Fx enums?

Reply:
these models are owned by this assembly model. It's a Power Apps owned model and implementation.
It's a dangerous thing to set serialization OM to a dependancy, as changes in the dependency may break logic here.
For this, it MAY be possible to create a shared implementation, but these OMs are designed for YamlDotNet usage, and PFx doesn't use that.

Reply:
not going to change this in this PR.